### DISCUSSION TITLE
can we find a job without a diploma

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#395] [General]
Hello, 
I learn solidity because I have lots of project ideas and I would like to know if I could integrate the job market, without a diploma.
If so, what personal project could I do for an interested hire?

**Comment by cromewar** [comment]
Hello there, yes you can get hired without a bachelor's degree or diploma (of course they help), If you want to get noticed and find job opportunities you could participate on hackathons, currently the chainlink hackathon is live and you still have time to submit a project.

I recommend you check the official web https://chain.link/hackathon and join the discord community (you'll find it on the link). Also take the full freecodecamp course, you will learn a lot. 

Best regards, Cromewar.

**Comment by wilonweb** [comment]
Thanks for the link
I saw a lot of profil With the different advocate, developer, CEO, marketing, Builder ... 
I'm "beta-junior" on backend, Do you know a good technology to learn to undertake web-applications like cryptopunk ?

**Reply by cromewar** [reply]
If you want to create NFTS fast i'll recommend you to check [Moralis](https://moralis.io/) and [this](https://www.youtube.com/watch?v=KBV4FrCv4ps) video.
